src/Run/Setup.cs

```diff
@@ -111,7 +153,7 @@ private string BuildParametersForCommand(Dictionary<string, string> commandParam
             {
                 if (!parameters.Key.Equals("toolName") && !string.IsNullOrEmpty(parameters.Value))
                 {
-                    string value = parameters.Value.Equals("default") ? FindSettingValue(parameters.Key) : parameters.Value;
+                    string value = parameters.Value.Equals("default") ? FindSettingValue(parameters.Key) : ParseSettingValue(parameters.Value);
```

Comment:
It is worth clarifying that `${}` is only valid for Settings value. It is not meant to be used in other parts of the config.json file as we don't want to have a pre-build config.json file step before actually using it.